src/source/feeds.rst

```diff
@@ -31,6 +82,701 @@ XML file.
+   sellside API.
+
+.. index:: sellerName
+.. _feed_sellerName:
+
+sellerName
+""""""""""
+
+Display name of the seller (max. 60 characters).
+
+======= ======================================================
+Example	.. code-block:: html
+        
+         <ad:sellerName>Cups, Caps &amp; Craps</ad:sellerName>
+======= ======================================================
+
+.. index:: title
+.. _feed_t:
+
+title
+"""""
+
+Any string, with minimum and maximum length determined by the category, with a maximum cap of 1024 characters. See :ref:`categories`. URLs are not allowed as part of the title.
+
+======= ====================================================
+Example	.. code-block:: html
+        
+         <ad:title>Goedkope A-merk herenfietsen</ad:title>
+======= ====================================================
+
+.. index:: description
+.. _feed_descr:
+
+description
+"""""""""""
+
+The description field of the ad.
+Any string, with minimum and maximum length determined by the category, with a maximum cap of 1MB characters. See :ref:`categories`. URLs are not allowed as part of the description.
+All HTML elements except for the ones below will be removed:
+
+.. code-block:: html
+
+    <u> <em> <ul> <li> <p> <strong> <br>
+    
+
+======= =========================================================================================================
+Example .. code-block:: html 
+    
+            <ad:description><![CDATA[
+                <p><strong><u>De goedkoopste webshop</u></strong> 
+                    <strong>voor tweedehands fietsen met garantie! Gratis en rijklaar thuisbezorgd!</strong>
+                </p>
+                <p><strong><br></strong>
+                </p>
+                <ul>
+                    <li><strong>Laagste prijsgarantie</strong></li>
+                    <li>Fietsen <strong>100% rijklaar</strong> gratis thuisbezorgd</li>
+                    <li><strong>Ruime voorraad</strong>, voor ieder wat wils</li>
+                    <li>Snelle <strong>customer service</strong> via Whatsapp, bellen en e-mail</li>
+                    <li>1 <strong>maand garantie</strong></li>
+                    <li>Aangesloten bij <strong>Webwinkelkeur</strong></li>
+                </ul>
+                <strong><br></strong>
+                <p>Check dus snel onze website en vind de fiets die bij je past!<br>
+                </p>
+                <strong><br></strong>
+                <p>WhatsApp, bel of mail ons voor verdere vragen.
+                </p>]]>
+            <ad:description/>
+======= =========================================================================================================
+
+
+.. index:: categoryId
+.. _feed_categoryId:
+
+categoryId
+""""""""""
+
+The category in which the ad is placed in the :ref:`categories` tree.
+
+Each ad belongs to one and only one category and the category of an ad cannot be changed.
+This field can only be set once during the lifecycle of an ad.
```

Comment:
we actually _do_ support changing of category.. I think it'll also be supported in feeds, could you check?

Reply:
This one was taken from the ads.rst, so if you doubt it, it is wrong for sure